### DISCUSSION TITLE
use validation=lower_case_extensions_without_dot for cleanup_list and unwanted_extensions

### DISCUSSION THREAD

**Discussion by sanderjo** [sabnzbd/sabnzbd#2139] [Ideas]
Easy first step: put validation=lower_case_extensions_without_dot  into sabnzbd/cfg.py in these lines

cleanup_list = OptionList("misc", "cleanup_list")
unwanted_extensions = OptionList("misc", "unwanted_extensions")

More difficult step: cleanup code that is not necessary anymore
https://github.com/sabnzbd/sabnzbd/blob/develop/sabnzbd/misc.py#L688-L692

I don't know where/if unwanted_extensions is cleaned up

**Comment by Safihre** [comment]
Implemented in 7b395784614babdac0c31da6be5cd4a547e4342d.
